iosApp/Modules/Screens/Home/UI/HomeView+Loaded.swift

```diff
@@ -23,7 +23,7 @@ extension HomeView {
 					BigCellSection(uiModel.recentlyPlayedSection, actionHandler: actionHandler.songTapped)
 					BigCellSection(uiModel.justReleasedSection, actionHandler: actionHandler.artistTapped)
 					BigCellSection(uiModel.moreOfWhatYouLikeSection, actionHandler: actionHandler.artistTapped)
-					ArtistsSection(uiModel.newmArtistsSection, actionHandler: actionHandler.artistTapped)
+                    ArtistsSection(uiModel.newmArtistsSection, actionHandler: actionHandler.artistTapped)
```

Comment:
These should be tabs.  Make sure your xcode preferences are set to tab indentation not spaces.